index.css

```diff
@@ -562,6 +563,117 @@ q:before, q:after{ content: "";}
     width: 100%;
 }
 
+/* section join */
+
+.section-join:before {
+    display: block;
+    content: "";
+    text-align: center;
+    height: 100px;
+    width: 2px;
+    margin: 0 auto;
+    background-color: #5da8ef;
+    margin-bottom: 63px;
+}
+
+.section-join {
+    margin: 0 auto;
+    width: 590px;
+    text-align: center;
+    margin-bottom: 148px;
+}
+
+.join__title {
+    color: #333333;
+    font-size: 42px;
+    font-weight: 700;
+    line-height: 66px;
+    margin: 0;
+}
+
+.join__text {
+    opacity: 0.65;
+    color: #616161;
+    font-size: 14px;
+    font-weight: 300;
+    line-height: 24px;
+    margin-bottom: 30px;
+}
+
+input[type="email"] {
+    width: 538px;
+    height: 48px;
+    background-color: #f5f5f5;
+    border: none;
+}
+
+::placeholder {
+    opacity: 0.9;
+    color: #333333;
+    font-family: "League Spartan";
+    font-size: 9px;
+    font-weight: 700;
+    text-transform: uppercase;
+    letter-spacing: 5px;
+    text-indent: 29px;
+}
```

Comment:
Note: this will style all placeholders across the website. I don't know if this is a desired behavior (probably it's okay if there is only 1 input).